src/styles/globals.scss

```diff
@@ -18,13 +18,13 @@
 @import "./elements/kbd";
 
 /* LAYOUT/OBJECTS: objects, grids, flexbox etc */
+@import "./components/component-grid";
 
 /* COMPONENTS: most of the work goes here! BEM classes, headers/footers/carousels, etc */
 @import "./components/home";
 @import "./components/details-page";
 @import "./components/header";
 @import "./components/component-card";
-@import "./components/component-grid";
 @import "./components/footer";
 @import "./components/layout";
```

Comment:
Should the layout file also be in the Layout section?

Reply:
Good call on that. I'll change it.